src/lib.rs

```diff
@@ -276,6 +276,8 @@ pub mod sequence;
 #[cfg(feature = "text")]
 pub mod text;
 
+mod arch;
```

Comment:
Should be moved to be with the other mod imports

Reply:
hey do you mind writing implementation for aarch64